### ISSUE TITLE
Road to v1.0

### ISSUE THREAD

**Issue by uzyn** [opauth/opauth#81]
v1.0 has been started for more than a year. 

@ceeram and I are now working towards the release of v1 of Opauth.

Some of the changes include:
- A complete rewrite from v0.x while retaining the core ideas of Opauth.
- No more internal callbacks.
- Cleaner code base and API
- PSR compliance
- and many more...
- To bring about these improvements, Opauth v1.x is not backward-compatible with v0.x.

Before we can hit the release of Opauth v1, here are still some tasks for us to complete:
- [ ] Updated user guide (documentations):
  - [x] Upgrading guide (consumer)
  - [x] Upgrading guide (strategy author)
  - [ ] Strategy contribution guide
- [ ] Updated API docs
- [ ] Updated web site for v1
- [ ] State support (test and document)
- [x] Update strategies maintained by Opauth organisation to v1
- [ ] Update community strategies from v0.x strategies to v1, submit PR for it and get all submitted to Packagist.
- [ ] Sample app (or this can simply be opauth.org's source)
- [ ] Better test coverage
#### Maybe / good to have
- [ ] Fully integrated web site with the following:
  - User guide
  - API docs
  - List of strategies that can be easily updatable
- [ ] Contact and/or update framework plugins for v1
- [ ] Base strategies for common auth protocols
#### Schedules
- [x] `1.0.0-alpha` released on 7 April 2014.
- stay tuned for next release schedule.
## Wanting to help?

Join us at IRC: #opauth on [Freenode](http://webchat.freenode.net/?channels=opauth&uio=d4)

**Comment by burzum** [comment]
is this project abandoned? 

There are just two commits since v1.0.0-alpha.1.

**Comment by ceeram** [comment]
Yup. Sorry
Op 16 aug. 2015 22:52 schreef "Florian Krämer" <notifications@github.com>:

**Comment by burzum** [comment]
@ceeram I read your "Yup" as "Yes, it has been abandoned". Meh... Why? Is there any alternative project? You should communicate on the main page of the project that it has been abandoned and point to an alternative or saying that you're looking for a maintainer.

**Comment by davidyell** [comment]
I agree with @burzum it would be nice to mention that it's no longer developed in the readme. Perhaps you could point to http://hybridauth.sourceforge.net/

**Comment by ceeram** [comment]
@burzum i see you think of writing a successor? how about adding you here as a maintainer instead?

**Comment by burzum** [comment]
@ceeram there is already a successor -> http://hybridauth.sourceforge.net/

And it would be fair and awesome to tell people that this project is abandoned, see https://github.com/opauth/opauth/issues/112#issuecomment-146530724

**Comment by ceeram** [comment]
Not my job. Hybridauth has been next to opauth for a long time already. I
assumed you prefered to not use that as alternative to opauth and rather
roll your own. Also phpleague has an oauth solution.
Op 8 okt. 2015 19:57 schreef "Florian Krämer" <notifications@github.com>:
